lib/dbservice/sessions/session.ex

```diff
@@ -36,6 +37,7 @@ defmodule Dbservice.Sessions.Session do
     many_to_many(:users, User, join_through: "user_session", on_replace: :delete)
     many_to_many(:group_type, GroupType, join_through: "group_session", on_replace: :delete)
     belongs_to(:form_schema, FormSchema)
+    has_many(:session_schedule, SessionSchedule)
```

Comment:
What is the use case for one session having multiple session schedules?

Reply:
If one session is happening for different batches and the schedule for each batch is different, then I think one to many will work